ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -32,9 +32,33 @@ Item {
     property real scrollY: chatLogView.visibleArea.yPosition * chatLogView.contentHeight
     property int newMessages: 0
 
+    Item {
+        id: loadintMessagesIndicator
```

Comment:
typo
```suggestion
        id: loadingMessagesIndicator
```

Reply:
@jrainville you sharp eye... you always find me some typo. 😄  I updated it, thank you.